gnocchi/carbonara.py

```diff
@@ -85,6 +84,42 @@ def round_timestamp(ts, freq):
         (ts - UNIX_UNIVERSAL_START64) / freq) * freq
 
 
+TIMESERIES_ARRAY_DTYPE = [('timestamps', 'datetime64[ns]'),
+                          ('values', 'float64')]
+
+
+def make_timeseries(timestamps, values):
```

Comment:
I thought about it but `make_timeseries` is used by `GroupedTimeSerie` and `TimeSerie` and they both have no ancestor in common.
Since it's a rather short function that just wraps Numpy for convenience, I decided to let it here, alone. :)

If we ever make `GroupedTimeSeries` inherits from `TimeSerie` (which may or may not make sense) we could do it. Thought it'd still be a `staticmethod` anyway. :)